Localization/localization.cfg

```diff
@@ -131,135 +131,135 @@ Localization
                 
-        precisemaneuver_timealarm_time                  = Time
-        precisemaneuver_timealarm_alarm                 = Alarm
-        precisemaneuver_timealarm_alarm_tooltip         = Add an alarm using the Kerbal Alarm Clock mod
+        precisemaneuver_timealarm_time                  = Tiempo
+        precisemaneuver_timealarm_alarm                 = Alarma
+        precisemaneuver_timealarm_alarm_tooltip         = Agregar alarma usando el mod Kerbal Alarm Clock
         
-        precisemaneuver_increment                       = Increment
+        precisemaneuver_increment                       = Incremento
         
-        precisemaneuver_orbittools                      = Orbit Tools
-        precisemaneuver_orbittools_turnup_tooltip       = Turn the orbit up
-        precisemaneuver_orbittools_turndown_tooltip     = Turn the orbit down
-        precisemaneuver_orbittools_circularize_tooltip  = Circularize the orbit with current maneuver
-        precisemaneuver_orbittools_copy_tooltip         = Copy maneuver details to clipboard
-        precisemaneuver_orbittools_paste_tooltip        = Paste maneuver from clipboard\n(you can copy it from the Transfer Window Planner mod)
+        precisemaneuver_orbittools                      = Herramientas de órbitas
+        precisemaneuver_orbittools_turnup_tooltip       = Agregar una órbita
+        precisemaneuver_orbittools_turndown_tooltip     = Reducir una órbita
+        precisemaneuver_orbittools_circularize_tooltip  = Circularizar la órbita con la maniobra actual
+        precisemaneuver_orbittools_copy_tooltip         = Copiar detalles de maniobra al copiapapeles
+        precisemaneuver_orbittools_paste_tooltip        = Copiar maniobra del copiapapeles\n(puedes copiarla desde el modTransfer Window Planner)
         
-        precisemaneuver_gizmo_undo                      = Undo
-        precisemaneuver_gizmo_redo                      = Redo
-        precisemaneuver_gizmo_sensitivity               = Sensitivity
+        precisemaneuver_gizmo_undo                      = Deshacer
+        precisemaneuver_gizmo_redo                      = Rehacer
+        precisemaneuver_gizmo_sensitivity               = Sensibilidad
         precisemaneuver_gizmo_Ap                        = Ap
         precisemaneuver_gizmo_Pe                        = Pe
                 
-        precisemaneuver_encounter                       = Next Enc.
+        precisemaneuver_encounter                       = Próximo Enc.
         precisemaneuver_encounter_PE                    = PE
                 
-        precisemaneuver_ejection_angle                  = Ej. Angle
-        precisemaneuver_ejection_angle_format           = ｢0:0.00° from prograde;0.00° from retrograde｣
-        precisemaneuver_ejection_inclination            = Ej. Incl.
-        precisemaneuver_ejection_inclination_format     = ｢0:0.00° north;0.00° south｣
+        precisemaneuver_ejection_angle                  = Ángulo de Eyec.
```

Comment:
Do not abbreviate 2 words in a row, it can't be understood, here should be "Ángulo de Ejec." or "Áng de Eyección" I prefer the first one.